src/SupervisionSuccess/components/Outcomes/Outcomes.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./Outcomes.css";
+
+const Outcomes = ({ prisonPopulationDiff, savings }) => {
+  const [prisonPopulationDiffText, prisonPopulationDiffIconClass] = (() => {
+    if (prisonPopulationDiff < 0) return ["Fewer people in prison", "outcomes_stat-icon-down"];
+    if (prisonPopulationDiff > 0) return ["More people in prison", "outcomes_stat-icon-up"];
+    return [null, ""];
+  })();
+
+  const [savingsText, savingsIconClass] = (() => {
+    if (savings > 0) return ["Reduced costs", "outcomes_stat-icon-down"];
+    if (savings < 0) return ["Increased costs", "outcomes_stat-icon-up"];
+    return [null, ""];
+  })();
+
+  return (
+    <div className="outcomes">
+      <div className="outcomes_heading">Outcomes</div>
+      <div className="outcomes_stats">
+        <div className="outcomes_stat">
+          <div className={`outcomes_stat-value ${prisonPopulationDiffIconClass}`}>
+            {Math.abs(prisonPopulationDiff)}
+          </div>
+          <div className="outcomes_stat-key">{prisonPopulationDiffText}</div>
+        </div>
+        <div className="outcomes_stat">
+          <div className={`outcomes_stat-value ${savingsIconClass}`}>{Math.abs(savings)}</div>
```

Comment:
The mocks look just about pixel-perfect! The only discrepancy I see is that this should be formatted with a dollar sign and a big M if the savings are in fact in the millions of dollars. Right now it's "32" but it should be either "$32" or "$32M" if it's millions.

Reply:
Ok, I've already done this logic:
If <10M - show 2 digits after dot with M, if <100M - show 1 digit after dot - else: show rounded integer. Is this ok? Or maybe some different rules should be applied?

Reply:
That sounds great!